WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/Create Shipping Label Form/Package Details/ShippingLabelPackageDetailsViewModel.swift

```diff
@@ -104,6 +123,44 @@ final class ShippingLabelPackageDetailsViewModel: ObservableObject {
     }
 }
 
+// MARK: - Package Selection
+extension ShippingLabelPackageDetailsViewModel {
+    func didSelectPackage(_ id: String?) {
+        guard id != nil else { return }
```

Comment:
Could you explain a bit on why is it possible to have nil `id`?

Reply:
Removed the optional `id` 👍